sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -203,7 +199,8 @@ public static void signOut() {
     public synchronized void onStarted(@NonNull Context context, @NonNull Channel channel, String appSecret, String transmissionTargetToken, boolean startedFromApp) {
         mContext = context;
         mAppSecret = appSecret;
-        mTokenStorage = TokenStorageFactory.getTokenStorage(context);
+        AuthTokenContext authTokenContext = AuthTokenContext.getInstance();
+        authTokenContext.setStorage(TokenStorageFactory.getTokenStorage(context));
```

Comment:
I don't like this implicit logic very much.. What will happen if a developer unfamiliar with this logic forgets to setStorage?
Maybe we revert to how it was where we just passed context to  AuthTokenContext.getInstance()?

Reply:
@annakocheshkova Token storage implemented in the identity module, context in core.
It was a conscious decision. No, getInstance didn't have context parameter